src/v2/Components/ArtworkFilter/CreateAlertButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react"
+import styled from "styled-components"
+import { themeGet } from "@styled-system/theme-get"
+import { BellIcon, Button, Flex, Spacer, Text } from "@artsy/palette"
+
+const StyledCreateAlertButton = styled(Button).attrs({
+  variant: "secondaryOutline",
+  size: "small",
+  width: 130,
```

Comment:
do we necessarily need to specify a fixed width here?

Reply:
I believe the answer is yes since we want it to be of the same width as in figma spec

![Screenshot 2021-11-03 at 13 30 55](https://user-images.githubusercontent.com/44819355/140044952-965293a0-75a5-4314-8301-d08f8ae3b43d.png)

Reply:
I am afraid that when using a fixed width, there may be problems with displaying the button label (for example, the user may have the page zoomed in the browser and the fixed width will not be enough for label). For example, as in the screenshot below, the button label goes beyond the offsets

![Image 2021-11-03 14-01-08](https://user-images.githubusercontent.com/3513494/140049236-f9244b60-1c4f-4ef8-952e-3773c8375a5b.png)